### ISSUE TITLE
Cannot get the build to fail with force:apex:test-run

### ISSUE THREAD

**Issue by afawcett** [forcedotcom/sfdx-travisci#58]
When i run the following from the command line on my workstation with failing tests in place...

~~~
sfdx force:apex:test:run --wait 10
~~~

And echo the return code i see the following...

~~~
echo $?
100
~~~
However when i run the same command (all be it with an alias arg) in Travis it returns 0?

![fail](https://user-images.githubusercontent.com/1167760/38774281-fa601e34-4019-11e8-968d-bb90529f26bc.png)

Is there something contextual about the commands return code?

You can see my project at https://github.com/afawcett/declarative-lookup-rollup-summaries

**Comment by afawcett** [comment]
[Take a look at this latest build for an example](https://travis-ci.org/afawcett/declarative-lookup-rollup-summaries/jobs/366671311).

**Comment by dcarroll** [comment]
Taking a look.  Please stand by.

**Comment by amphro** [comment]
The only conditional is that the `failing` property is set and not 0. You could run the command with `--json` and look at that property in the summary. Something like `... | jq .result.summary.failing`. I'm not sure how that would happen, but if there is no summary or something else looks off, that could explain it.

**Comment by dcarroll** [comment]
Ah, --json might be the key, I'll test it out.

**Comment by afawcett** [comment]
@amphro yeah the really odd thing is i get a 100 status code when running locally. This repo however does seem to work, yet @dcarroll and i can see now difference between my repos [yaml](https://raw.githubusercontent.com/afawcett/declarative-lookup-rollup-summaries/master/.travis.yml) config and this [one](https://raw.githubusercontent.com/forcedotcom/sfdx-travisci/master/.travis.yml) (that is working). So i would say the output could be the difference if it was not for the fact it outputs the correct status locally... its a complete mystery.... i think i will try CircleCI next...

**Comment by afawcett** [comment]
@amphro and thanks for the jq idea... i will keep that in mind, cool tool!

**Comment by jcory** [comment]
@afawcett @dcarroll Did you ever reach any sort of resolution for this? We're encountering the same issue. I'd like to avoid switching to CircleCI, but at this point, that may be our best option.

**Comment by christianmoustrides** [comment]
Anyone figure this one out?

**Comment by ccheevers** [comment]
Any update on this issue, a failing assertion still results in a passed test with travis....
